src/jep/convert_p2j.c

```diff
@@ -41,6 +42,9 @@
 
 #define JCHAR_MAX   0xFFFF
 
+static jmethodID classGetMethods = 0;
+static jmethodID classGetModifiers = 0;
+static jmethodID methodGetModifiers = 0;
```

Comment:
I've actually got a [commit saved off](https://github.com/bsteffensmeier/jep/commit/6081a07d7f7c2c071b2849180ceb56b74e8695ac) that I intend to merge into 3.7 that consolidates all of the java method calls. I just haven't gotten around to merging it yet. So for this review I recommend we just leave all the duplicates and I will merge my change after this goes in and clean up this with everything else.